### PR TITLE
[SGP-6170] Fixes missing `size` attribute error.

### PR DESCRIPTION
**Pull request by rbevers**
- Adds a `size` attribute to `FileObject` to fulfill expectations of the Mezzanine 4.3.1 blog feed.
- Adds a minimal set of unit tests to verify this addition is equivalent to `filesize`.

### CODE REVIEW

**Comment by rbevers** [comment]
Reproducing test case:

```
======================================================================
ERROR: test_size_is_filesize (filebrowser_safe.test_base.FileObjectTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/rbevers/Work/bevy/filebrowser-safe/filebrowser_safe/test_base.py", line 13, in test_size_is_filesize
    self.assertEqual(fileobj.size, 17834)
AttributeError: FileObject instance has no attribute 'size'
```

**Comment by rbevers** [comment]
To test locally, I did the following:

```
$ ./test.sh
# Runs Mezzanine tests with this filebrowser version installed.
# ... lots of messy output, but no failures...
$ python setup.py tests
running test
running egg_info
writing filebrowser_safe.egg-info/PKG-INFO
writing top-level names to filebrowser_safe.egg-info/top_level.txt
writing dependency_links to filebrowser_safe.egg-info/dependency_links.txt
reading manifest file 'filebrowser_safe.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
writing manifest file 'filebrowser_safe.egg-info/SOURCES.txt'
running build_ext
test_size_is_filesize (filebrowser_safe.test_base.FileObjectTests) ... ok

----------------------------------------------------------------------
Ran 1 test in 0.032s

OK
```

**Comment by rbevers** [comment]
@abendig This has a CircleCI config, but won't let me configure it...  "Contact repo admin"

screen shot 2019-02-25 at 3 09 39 pm

**Comment by rbevers** [comment]
@abendig Ping.

**Comment by rbevers** [comment]
Should we also have Chris in this repo? Assigned to you because no one else is here.

**Review by abendig — APPROVED** [review]
The changes look good to me.

I set up following on CI and added Chris to the team and this PR.

**Comment by rbevers** [comment]
@ChrisAikman This repo may be unfamiliar. And it's not notifying Slack. Hope you see this.

File: setup.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 setup(
     name="filebrowser_safe",
-    version="0.5.0bevy2",
+    version="0.5.0.post3",
```
Comment (ChrisAikman) [inline setup.py:6]: Curious: what is this version convention based on?

**Review by ChrisAikman — CHANGES_REQUESTED** [review]
Looks good! I have one question for my curiosity. :+1: :shipit:

**Comment by ChrisAikman** [comment]
Sorry, one more question. It seems the forked library is no longer maintained. Is the idea to keep patching the library as needed, or to eventually look elsewhere?

File: setup.py (hunk 1 above)
Comment (rbevers) [reply setup.py:6]: I learned, in trying to make the warnings about wrong dependency versions for Mezzanine go away, that Python packages have their own versioning scheme, specified in [PEP 440](https://www.python.org/dev/peps/pep-0440/#post-releases).

Based on experimentation, a post release version was the only way to increment the version, but also satisfy the dependency on `>=0.5.0` without a likely collision with a real-live release version. I wanted to get `bevy` in the version string but it gave warnings every time. Perhaps I didn't read the PEP carefully enough...

**Comment by rbevers** [comment]
Not sure what led you to think it is no longer maintained. It doesn't get changes often. *And* we've forked the version that the Mezzanine guy forked from the original project. So it is a bit unclear to me **how** maintained it is.

That said, we are operating on an older version here because of the speed-up necessary for our Softlayer storage, so technically our `0.5.0.post3` is actually a flavor of `0.4.3` we've modified. The version fake is to make Mezzanine 4.3.1 happy.

**Review by ChrisAikman — APPROVED** [review]
Sorry, somehow clicked `Request changes` instead of `Approve`. For real this time: :+1: :shipit:

**Pull request merged**